### PR TITLE
docs:(PPDSC-2709) update icons scenarios in storybook

### PR DESCRIPTION
**Pull request by LukeFinch**
PPDSC-2709

**What**

1. Background - why this is needed
Update the storybook scenarios for icons.
3. What did you do
4. What does the reviewers should expect

**I have done:**
- [ ] Written unit tests against changes
- [ ] Written functional tests against the component and/or NewsKit site
- [ ] Updated relevant documentation

**I have tested manually:**
- [ ] The feature's functionality is working as expected on Chrome, Firefox, Safari and Edge
- [ ] The screen reader reads and flows through the elements as expected.
- [ ] There are no new errors in the browser console coming from this PR.
- [ ] When visual test is not added, it renders correctly on different browsers and mobile viewports (Safari, Firefox, small mobile viewport, tablet)
- [ ] The Playground feature is working as expected

**Before:**

**After:**

**Who should review this PR:**

**How to test:**

### CODE REVIEW

**Comment by pp-serviceaccount** [comment]
You can preview these changes on:
  Documentation Site: http://ncu-newskit-docs-pr.s3-website-eu-west-1.amazonaws.com/ppdsc-2709-update-icons-scenarios-storybook
  Storybook: http://ncu-newskit-docs-pr.s3-website-eu-west-1.amazonaws.com/ppdsc-2709-update-icons-scenarios-storybook/storybook

File: yarn.lock [hunk 1]
```diff
@@ -1431,13 +1431,20 @@
     core-js-pure "^3.19.0"
     regenerator-runtime "^0.13.4"
 
-"@babel/runtime@^7.0.0", "@babel/runtime@^7.10.2", "@babel/runtime@^7.12.13", "@babel/runtime@^7.12.5", "@babel/runtime@^7.14.0", "@babel/runtime@^7.14.8", "@babel/runtime@^7.15.4", "@babel/runtime@^7.16.3", "@babel/runtime@^7.3.1", "@babel/runtime@^7.4.4", "@babel/runtime@^7.5.0", "@babel/runtime@^7.5.5", "@babel/runtime@^7.7.6", "@babel/runtime@^7.8.4", "@babel/runtime@^7.8.7", "@babel/runtime@^7.9.2":
+"@babel/runtime@^7.0.0", "@babel/runtime@^7.10.2", "@babel/runtime@^7.14.0", "@babel/runtime@^7.14.8", "@babel/runtime@^7.15.4", "@babel/runtime@^7.16.3", "@babel/runtime@^7.3.1", "@babel/runtime@^7.4.4", "@babel/runtime@^7.5.0", "@babel/runtime@^7.5.5", "@babel/runtime@^7.7.6", "@babel/runtime@^7.8.4", "@babel/runtime@^7.8.7", "@babel/runtime@^7.9.2":
```
Comment (mutebg) [inline yarn.lock:1434]: Why is this changed?

File: src/icons/__tests__/icons.stories.tsx [hunk 2]
```diff
@@ -53,185 +33,156 @@ const iconsCustomThemeObject: CreateThemeArgs = {
...
+          size: 'iconSize050',
+        }}
+      />
+    </StorybookCase>
+  </StorybookPage>
+);
+StorySizeIcons.storyName = 'Size';
 
 export const StoryCustomIcons = () => (
   <>
```
Comment (mutebg) [inline src/icons/__tests__/icons.stories.tsx:122]: I think you don't need this fragment here

**Review by mutebg — APPROVED** [review]
